config/schema.error.yaml

```diff
@@ -0,0 +1,567 @@
+2:
+    title: 'Cannot write to the `./app/etc/env.php` file'
+    suggestion: 'Deployment script cannot make required changes to the `/app/etc/env.php` file. Check your filesystem permissions.'
+    stage: build
+    type: critical
+3:
+    title: 'Configuration isn''t defined in the `schema.yaml` file'
+    suggestion: 'Configuration isn''t defined in the `./vendor/magento/ece-tools/config/schema.yaml` file. Check that the config variable name is correct, and that it defined.'
```

Comment:
Actually this is a good example when better to read title from Exception, because there we may add name of variable which otherwise it is very abstract suggestion.
So, probably add additional option which says to write original message from exception?

Reply:
Suggestion and title which are passed to logger have higher priority then title and suggestion from `schema.error.yaml`. So it already works in the way you described.
In case if someone runs a command to get error info `error:show 3` title and suggestion will be taken from `schema.error.yaml`.